Repository/ServiceEntityRepository.php

```diff
@@ -28,6 +28,13 @@ public function __construct(ManagerRegistry $registry, $entityClass)
     {
         $manager = $registry->getManagerForClass($entityClass);
 
+        if (null === $manager) {
+            throw new \LogicException(sprintf(
+                'Could not find the entity manager for class "%s". Did you forget to configure doctrine where to find the entity mapping?',
```

Comment:
How about:

... Did you forget -> Check your Doctrine configuration to make sure it is configured to load this entity’s metadata.

Reply:
done